src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php

```diff
@@ -28,7 +28,7 @@
      * Add or import routes into your application.
      *
      *     $routes->import('config/routing.yml');
-     *     $routes->add('/admin', 'AppBundle:Admin:dashboard', 'admin_dashboard');
+     *     $routes->add('/admin', 'App\Controller\AdminController::dashboard', 'admin_dashboard');
```

Comment:
Shouldn't this be a single colon now?

Reply:
didn't we deprecate single colon syntax?

Reply:
Here's were it happened https://github.com/symfony/symfony/pull/26085
So if I got it right, we now have `Class::method` instead of `a:b:c` and `a:b`

Reply:
deprecation for OAuth bundle says the same
![image](https://user-images.githubusercontent.com/2552865/42373599-1793894c-811e-11e8-98da-9011d5d45426.png)

Reply:
indeed, I confused that